README.rst

```diff
@@ -48,12 +46,10 @@ Docker usage
 
 If you do not want to install any additional libraries/dependencies you can use the Docker image.
 
-Example for external web site check:
-```
-docker run --rm -it -u $(id -u):$(id -g) linkchecker/linkchecker --verbose https://google.com
-```
+Example for external web site check::
+
+  docker run --rm -it -u $(id -u):$(id -g) linkchecker/linkchecker --verbose https://google.com
```

Comment:
Unrelated to this PR, but I strongly feel that we should be using IANA-approved example URLs.
```suggestion
  docker run --rm -it -u $(id -u):$(id -g) linkchecker/linkchecker --verbose https://example.com
```

Reply:
Done and upgraded the example.com above to https.